devices/philips.js

```diff
@@ -59,72 +92,20 @@ const hueExtend = {
                 .withLengthMin(1)
                 .withLengthMax(9)
                 .withDescription('List of RGB HEX colors'),
+            exposes.enum('effect', ea.SET, [
```

Comment:
I think effect is added twice here because `exposes: extendDontUse.light_onoff_brightness_colortemp_color({supportsHS: true, ...options}).exposes.concat([` is called without `disableEffect`

Reply:
Yeah looks like you're right. I've fixed it in the new commit.